selftests/utils.py

```diff
@@ -146,6 +148,22 @@ def skipUnlessPathExists(path):
     )
 
 
+def missing_binary(binary):
```

Comment:
This utility is already imported in vmimage testing refactor in https://github.com/avocado-framework/avocado/commit/7937e42ad6470c96a077bfedf2a7cd9b31eeae21 , am I right? Therefore, I would move it to that commit.